modules/apps/foundation/users-admin/users-admin-api/src/main/resources/content/Language.properties

```diff
@@ -1 +1,9 @@
-user.file.uploads.configuration.name=User Images
\ No newline at end of file
+check-token=Check image token
+maximum-height=Maximum height
+maximum-width=Maximum width
+Set the maximum file size for user portraits. A value of 0 for the maximum file size can be used to indicate unlimited file size. However, the maximum file size allowed is set in the property "com.liferay.portal.upload.UploadServletRequestImpl.max.size".
```

Comment:
Remove this extra line. This is the message for `users-image-maximum-file-size-help` at line 7